lca-main.yaml

```diff
@@ -44,6 +44,15 @@ Parameters:
       Processor Lambda. Enabling Amazon Chime SDK Call Analytics also allows you to also use Voice Analytics, 
       such as Voice Tone Analysis and Speaker Search.
 
+  CustomVoiceConnectorId:
+    Type: String
+    Default: ''
+    MaxLength: 50
```

Comment:
Too short..  My Voice ConnectorId name is > 50 characters..

![image](https://github.com/aws-samples/amazon-transcribe-live-call-analytics/assets/10953374/9b580d05-38cf-4670-8477-3d27f5ebaea6)

![image](https://github.com/aws-samples/amazon-transcribe-live-call-analytics/assets/10953374/51df6e25-5ae3-455d-89cb-3aee00b1d690)

Reply:
Will update the description to make it clear that it is the Voice Connector ID and not the Voice Connector Name.